doc/upgrade/7.2.md

```diff
@@ -4,9 +4,18 @@ See `doc/bc/changes-7.2.md` for requirements changes and deprecations.
 
 ## MySQL/MariaDB database tables character set change
 
-The character set for MySQL/MariaDB database tables is changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql` for the SQL upgrade statements.
+The character set for MySQL/MariaDB database tables should be changed from `utf8` to `utf8mb4` to support 4-byte characters. See `data/update/mysql/dbupdate-7.1.0-to-7.2.0.sql` for the statements that shorten table indexes, these must be run first. Beware that these upgrade statements may fail due to index collisions, which may occur when indexes are shortened. If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
 
-Beware that these upgrade statements may fail due to index collisions. This is because the indexes have been shortened, so duplicates may occur. If that happens, you must remove the duplicates manually, and then repeat the statements that failed.
+After successfully running those statements, change the character set of each table to `utf8mb4`. This can be done as shown below, for each of the standard tables in eZ Platform, and for any custom table you may want to convert. You may specify a collation; if you don't, the default collation for utf8mb4 will be used.
+
+To get a list of tables, enter the SQL command `show tables;`. For each of the tables you want to convert, replace `tbl_name` with the name of the table to convert:
+```sql
+-- Example using default collation
+ALTER TABLE tbl_name CONVERT TO CHARACTER SET utf8mb4;
```

Comment:
@glye Do we need to show an example setting default collation when we really want users to specify `utf8mb4_unicode_520_ci`?

Reply:
@alongosz Yeah, I can drop that. I should add a bit more info about collation, too.